WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldEditorView.swift

```diff
@@ -197,9 +209,15 @@ private extension CustomFieldEditorView {
             value: "HTML",
             comment: "Picker option for using Text Editor"
         )
+
+        static let deleteButton = NSLocalizedString(
+            "customFieldEditorView.deleteButton",
+            value: "Delete Custom Field",
```

Comment:
Super nit: I think we don't need to capitalize every word for the button.

```suggestion
            value: "Delete custom field",
```